src/libclient/controller/reportcontroller.cpp

```diff
@@ -172,6 +177,24 @@ void ReportController::Private::onTimingChanged()
               .arg(timer.timing()->timeLeft() / 1000));
 }
 
+void ReportController::Private::rotate()
+{
+    QRegExp regex("^\\d+_(\\d{4}-\\d{2}-\\d{2}).json$");
```

Comment:
can you add a comment to this line (maybe with an example what the regex matches)